.eslintrc.js

```diff
@@ -0,0 +1,55 @@
+module.exports = {
+  "root": true,
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parserOptions": {
+    "sourceType": "module",
+    "allowImportExportEverywhere": true
+  },
+  "extends": "airbnb",
+  "globals": {
```

Comment:
Thank you very much Chuks for this great implementation, I have seen so much from this code that I do not know about previously. But I'm curious as to what this `globals` property does. can you fill me in?

Reply:
According to the [ESLint configuration page](https://eslint.org/docs/user-guide/configuring)
There are several pieces of information that can be configured:

Environments - which environments your script is designed to run in. Each environment brings with it a certain set of predefined global variables.
Globals - the additional global variables your script accesses during execution.
Rules - which rules are enabled and at what error level.
All of these options give you fine-grained control over how ESLint treats your code.